service/controller/v13/resource/encryptionkey/delete_test.go

```diff
@@ -5,9 +5,8 @@ import (
 	"testing"
 
 	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
-	"github.com/giantswarm/micrologger/microloggertest"
-
 	"github.com/giantswarm/aws-operator/service/controller/v13/encrypter"
+	"github.com/giantswarm/micrologger/microloggertest"
```

Comment:
The blocks are weird here as well. Below as well.

Reply:
Indeed, fixed.